modules/nf-core/taxpasta/merge/meta.yml

```diff
@@ -25,14 +25,18 @@ input:
       type: file
       description: A list of taxonomic profiler output files (typically in text format, mandatory)
       pattern: "*.{tsv,csv,arrow,parquet,biom}"
-  - samplesheet:
-      type: file
-      description: A samplesheet describing the sample name and a filepath to a taxonomic abundance profile that needs to be relative from the work environment. The profiles must be provided even if you give a samplesheet as argument (optional)
-      pattern: "*.{tsv,csv,ods,xlsx,arrow,parquet}"
+  - profiler:
+      type: string
+      description: Name of the profiler used to generate the profile (mandatory)
+      pattern: "bracken|centrifuge|diamond|ganon|kaiju|kmcp|kraken2|krakenuniq|megan6|metaphlan|motus"
   - taxonomy:
       type: directory
       description: Directory containing at a minimum nodes.dmp and names.dmp files (optional)
       pattern: "*/"
+  - samplesheet:
+      type: file
+      description: A samplesheet describing the sample name and a filepath to a taxonomic abundance profile that needs to be relative from the work environment. The profiles must be provided even if you give a samplesheet as argument (optional)
```

Comment:
The paths in the sample sheet have to be defined based on wherever the files are, which in the case of nextflow is the `work` directory of the process. 

I'm not too concerned about the phrasing because most people will not use that channel, and if they do they will understand what it means 😬